Resources/Prototypes/Catalog/Fills/Boxes/general.yml

```diff
@@ -133,6 +133,22 @@
       components:
       - IdCard
 
+- type: entity
+  name: headset box
+  parent: BoxCardboard
+  id: BoxHeadset
+  description: A box of spare passenger headsets.
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingHeadsetGrey
+        amount: 6
+  - type: Sprite
+    layers:
+      - state: box
+      - state: headset
+  - type: Storage
+    capacity: 60
```

Comment:
In here, you need to add a whitelist, just like the other boxes have.
Your pr is currently failing the StorageTest test since the parent, "BoxCardboard", has an item size of 31, but in here, you're giving it a storage capacity of 60.

Basically, you've allowed this box to store 60 units, while itself only weighs 31. This allows you to create infinite storage, basically.

So you can either just add a whitelist of headsets to this box, or change the Item size itself to 60.

Reply:
Ah crap, you're right. No clue how I overlooked that.